cmd/vhs/main.go

```diff
@@ -86,6 +87,8 @@ func newRootCmd() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&cfg.ProfilePathMemory, "profile-path-memory", "", "Output memory profile to this path.")
 	cmd.PersistentFlags().StringVar(&cfg.ProfileHTTPAddr, "profile-http-address", "", "Expose profile data on this address.")
 
+	cmd.PersistentFlags().StringVar(&cfg.Plugin, "plugin", "", "Path to plugin shared object.")
```

Comment:
This only enables the loading of a single plugin object, correct? Is there value in supporting loading multiple plugin objects? (i.e. change this to a `StringSlice` instead of a `StringVar` and iterating over it below to load each of the specified plugins in turn.)

Reply:
Correct, we only support a single `*.so` file but that shared object can have as many components in it as the user wants.

Reply:
But what if people in the community start contributing components, and I want to use a source from one contributor and a sink from another contributor, for example?

Reply:
That's a good thought, I hadn't considered that. I will have to rethink the API a bit.

Reply:
I will merge this in now and rethink multiple plugins in a new PR, thanks for bringing this up.

Reply:
No problem, happy to ~~throw a wrench into the works~~ help! :grin: